ts/editor/rich-text-input/RichTextInput.svelte

```diff
@@ -253,5 +253,9 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
         /* Adopts same size as the content editable element */
         width: 100%;
         height: 100%;
+        /* Keep text on single line and hide overflow */
+        white-space: nowrap;
```

Comment:
Why would we want to restrict the placeholder to a single line though?

Reply:
If the description is super long and expands the field to multiple lines, it's an awkward experience when it collapses to one line when the user starts typing.

The input field in the field description is very short as well, already nudging users towards shorter descriptions.

The feature - at least how I understood it - is intended for short, concise descriptions.

Reply:
I see. I didn't consider that the placeholder would actually make the editor field grow bigger than a single line...